blacklist/blacklist/Scenes/Home/HomeViewController.swift

```diff
@@ -93,12 +102,65 @@ extension HomeViewController: UITableViewDataSource, UITableViewDelegate {
         }
     }
 
+    func addTableViewPlaceholderView() {
+        let header = "Noting so far" // TODO: Localize this string
+        let subheader = "You are on day :)" // TODO: Localize this string
+
+        let view = UIView(frame: .zero)
+
+        let headerLabel = UILabel(frame: .zero)
+        let subheaderLabel = UILabel(frame: .zero)
+
+        headerLabel.text = header
+        headerLabel.textAlignment = .center
+        headerLabel.font = UIFont(name: StyleSheet.Font.robotoRegular, size: Constants.emptyViewHeaderSize)
+        headerLabel.textColor = UIColor(hex: 0x222831)
+
+        subheaderLabel.text = subheader
+        subheaderLabel.textAlignment = .center
+        subheaderLabel.font = UIFont(name: StyleSheet.Font.robotoLight, size: Constants.emptyViewSubHeaderSize)
+        subheaderLabel.textColor = UIColor(hex: 0x222831)
```

Comment:
Same here, the color should be on the constants file.

Reply:
You're right, I'll move it